src/app/directives/url/url-active.directive.ts

```diff
@@ -35,9 +35,8 @@ export class UrlActiveDirective extends RouterLinkActive {
     _element: ElementRef,
     _renderer: Renderer2,
     _cdr: ChangeDetectorRef,
-    @Optional() _link?: RouterLink,
     @Optional() _linkWithHref?: UrlDirective
   ) {
-    super(_router, _element, _renderer, _cdr, _link, _linkWithHref);
+    super(_router, _element, _renderer, _cdr, _linkWithHref);
```

Comment:
Inconsistent parameter naming compared to https://github.com/QutEcoacoustics/workbench-client/pull/2085/files#diff-6c89349913cac853641bb63226dabd1f09947f41a4711a4306a400d5e2f9f686L47